airgun/entities/cloud_insights.py

```diff
@@ -48,6 +48,40 @@ def update(self, values):
         view = self.navigate_to(self, 'All')
         view.fill(values)
 
+class RecommendationsTabEntity(BaseEntity):
+    endpoint_path = '/foreman_rh_cloud/insights_cloud'
+
+    def search(self, value):
+        """Search for 'query' and return matched hostnames/recommendations.
+
+        :param value: text to filter (default: no filter)
+        """
+        view = self.navigate_to(self, 'All')
+        view.clear_button.click()
+        view.search_field.fill(value)
+        return self.table.read()
+
+    def affected_systems(self, recommendation_name: str):
```

Comment:
```suggestion
    def open_affected_systems(self, recommendation_name: str):
```
I would name the entity like this ^^^ so it is clear from the name what it does.